src/table/table.scss

```diff
@@ -16,75 +15,203 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-header {
+    @include iui-table-header;
+  }
+
+  .iui-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  &.iui-zebra-striping > .iui-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-body {
+    max-height: $iui-baseline * 27;
```

Comment:
Somehow this makes less sense than 300px 😄

Reply:
LOL I do agree, I switched it to a variable that equaled something as close to our original value of 300.
The `max-height` should be defined by the user, not us.  So then maybe the `max-height` should be set within `table.html` instead of the SCSS?

Reply:
Yeah, I think so too. But then what does this class even give us?

Reply:
Shouldn't Fixed has multiple set size instead of one default size ? small, default, large ?

Reply:
It's really only applying overflow related stuff to the body of the table.  I see how this might be similar to our overflow discussions we had with the tag container component.

@maxime4000, I'd rather let the user set a custom height for the fixed table.  I'm imagining the fixed table will be placed in areas with other UI surrounding it and we'd only be guessing what sizes might be best for them.

Reply:
Yeah I prefer that too. fixed should set the style for a set height but do not set the height directly and leave that to the dev.

Reply:
See https://github.com/iTwin/iTwinUI-react/issues/72 for a live example of how users are experiencing this.

Maybe we should just set overflow by default?

We can then use the fixed class to implement an actual fixed header (which stays on top of page even when scrolled down). To see an example, go to any file in a github repo and start scrolling down. You'll see that the filename bar stays sticky on top.